src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,129 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private Object[] elements;
+    private int size;
+
+    public ArrayList() {
+        this.size = 0;
+        this.elements = new Object[size];
```

Comment:
Your code is not implemented according to the task. 
![image](https://user-images.githubusercontent.com/20326858/221813611-9ea82482-eb5c-4fb7-bd5a-bbcc7340b507.png)

Reply:
+